games/0.yaml

```diff
@@ -71,8 +71,8 @@
   originals:
   - Runescape Classic
   type: remake
-  repo: https://github.com/2006rebotted/2006rebotted
-  url: https://rsrebotted.com/
+  repo: https://github.com/2006-Scape/2006Scape
```

Comment:
need to rename this game to 2006Scape too

Reply:
Sure